setup.cfg

```
@@ -23,7 +23,7 @@ install_requires =
 	pywin32-ctypes!=0.1.0,!=0.1.1; sys_platform=="win32"
 	SecretStorage>=3.2; sys_platform=="linux"
 	jeepney>=0.4.2; sys_platform=="linux"
-	importlib_metadata >= 3.6
+	importlib_metadata < 3.10; python_version < "3.10"
```

We should still specify the minimum version of `importlib_metadata`, and I don't see the point of specifying the maximum version (the latest version of backport is 4.x and it should work fine).

---

I was just trying to solve the open issue. I'm not super familiar with this. What I got was that importlib_metadata was kind of increasing the dependencies that may not be required for python versions over 3.10. 
If you want I can put importlib_metadata to be compolusory for python >=3.6 and <= 3.10. 
Hope this works out! Just starting out with open source so hoping it's fine.

---

The correct line would be:
```suggestion
	importlib_metadata >= 3.6; python_version < "3.10"
```

This means: require `importlib_metadata` at least 3.6 for Python older than 3.10.

---

Oh I understand now! Thank you! I'll commit this suggestion so that it's added to the pull request!
thanks for the tip. First time in open source so was really lost!
Appreciate it!

---

Now please also see this: https://github.com/jaraco/keyring/pull/576#pullrequestreview-987281982

It needs to be replaced in three files: `hook-keyring.backend.py`, `keyring/backend.py`, `tests/test_packaging.py`.